rfd/0131-adminitrative-actions-mfa.md

```diff
@@ -0,0 +1,260 @@
+---
+authors: Brian Joerger (bjoerger@goteleport.com)
+state: draft
+---
+
+# RFD 131 - Administrative Actions MFA
+
+## Required Approvers
+
+- Engineering: @r0mant && @codingllama
+- Product: @klizhentas || @xinding33
+
+## What
+
+Enforce MFA verification for any administrative actions taken. For example:
+
+- Adding, removing, or editing Nodes and other services.
+- Adding, removing, or editing users, roles, auth connectors, and other
+administrative resources.
+- Approving access requests.
```

Comment:
Can we list a few examples of actions that are **not** considered administrative actions since this list is so exhaustive? I think it might help illustrate what's in scope and what isn't.

Edit: as I read further I saw the examples. This is probably sufficient.